src/app/shared/models/psat.ts

```diff
@@ -13,6 +13,7 @@ export interface PSAT {
   operatingHours?: OperatingHours;
   valid?: PsatValid;
   existingDataUnits?: string;
+  whatIfScenario?: boolean;
```

Comment:
Put the whatIfScenario property on PsatInputs instead of PSAT, so you don't need to pass it in to psat-report -> getResults(). Same with field-data-service.ts or anywhere else where you've added it as an argument and we already have PsatInputs. In Field-data.ts you won't need to pass in whatIfScenario to get the form... this.baseline will be accurate since it will be based off the whatIfScenario passed from the parent.